utils.go

```diff
@@ -23,10 +22,10 @@ func ToString(i interface{}) interface{} {
 	if(i == nil) {
 		return "null"
 	}
-	v := reflect.ValueOf(i)
-	switch reflect.TypeOf(i).Kind() {
-	case reflect.String:
-		s := v.String()
+
+	switch i.(type) {
+	case string:
+		s := i.(string)
 		if len(s) == 0 {
```

Comment:
Shouldn't we wrap s in `" s "` regardless if s is: `"..."` e.g. s value contains `"`